scripts/deploy.sh

```diff
@@ -67,6 +70,7 @@ fi
 which openstack-ansible || ./scripts/bootstrap-ansible.sh
 
 # ensure all needed passwords and tokens are generated
+./scripts/pw-token-gen.py --file /etc/openstack_deploy/user_secrets.yml
```

Comment:
This definitely needs to go into the DEPLOY_AIO block, and more specifically the DEPLOY_CEPH block.  @git-harry, I'm a bit confused as to why it wouldn't be needed though.  The initial pw-token-gen.py is run in bootstrap-aio.sh, and we then drop in additional secrets and therefore need to re-run pw-token-gen.py.